srcpkgs/gptfdisk/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gptfdisk'
 pkgname=gptfdisk
-version=1.0.6
+version=1.0.8
 revision=1
 build_style=gnu-makefile
```

Comment:
```suggestion
build_style=gnu-makefile
make_check_target=test
```

Reply:
Just a suggestion if you want to run the tests

Reply:
Done.